risc0/bigint2/src/field/mod.rs

```diff
@@ -182,7 +182,9 @@ pub fn modsub_384_unchecked(
 
 // These "checked" versions verify that `result < modulus`
 
-pub fn modadd_256(
+#[stability::unstable]
+#[no_mangle]
+pub extern "C" fn modadd_256(
```

Comment:
I suggest we prefix extern "C" calls to make them globally unique.

```suggestion
pub extern "C" fn risc0_bigint_modadd_256(
```

Reply:
Ok, but then it won't work with patched crates (that are pure Rust) that are already using these functions, such as bn254
An alternative is to duplicate these functions (that is, have one version that's extern "C", and another version that's just for rust, where the body of these two versions is the same). @SchmErik also looked at the current approach and so far it seemed ok to him.

Reply:
This will break users downstream if the symbol names are not unique enough. 

Extern "C" functions need to be globally unique. You can have the extern "C" functions call the rust functions, that would be fine.

Reply:
Ok, I changed the extern "C" functions to use risc0_bigint_ prefix.
@tzerrell I did it in the risc0-bigint2 crate as you recommended (and not in zkvm-platform), though as mentioned the setup for 3rd-party crates will be more involved.